packages/react-components/src/components/Select/SelectList.tsx

```diff
@@ -0,0 +1,202 @@
+import * as React from 'react';
+import cx from 'classnames';
+import * as MaterialIcons from '@livechat/design-system-icons/dist/material';
+import { ISelectItem } from './interfaces';
+import { EventKeys } from '../constants';
+import Icon from '../Icon';
+
+const baseClass = 'lc-select-body';
+
+interface IHoverCallbacksProps {
+  [key: string]: () => void;
+}
+
+export interface ISelectListProps {
+  isOpen?: boolean;
+  items: ISelectItem[];
+  selectedItem?: string | number | null;
+  focusedItemKey: string | null;
+  listRef: React.RefObject<HTMLUListElement>;
+  selectHeader?: string;
+  getItemBody: (props: ISelectItem['props']) => React.ReactNode;
+  onListClose: () => void;
+  getItemSelectedHandler: (itemKey: string) => void;
+  onEnterKey: (itemKey: string) => void;
+  onFocusedItemChange: (itemKey: string) => void;
+}
+
+export const SelectList: React.FC<ISelectListProps> = ({
+  isOpen,
+  items,
+  selectedItem,
+  focusedItemKey,
+  listRef,
+  selectHeader,
+  getItemBody,
+  onListClose,
+  getItemSelectedHandler,
+  onEnterKey,
+  onFocusedItemChange,
+}) => {
+  const hoverCallbacks: IHoverCallbacksProps = {};
+  let timerId: ReturnType<typeof setTimeout> | null = null;
+
+  React.useEffect(() => {
+    if (isOpen) {
+      if (listRef.current) {
+        listRef.current.scrollTop = 0;
+      }
```

Comment:
```suggestion
        listRef?.current.scrollTop = 0;
```

Reply:
I didn't change that because:
`The left-hand side of an assignment expression may not be an optional property access.ts(2779)`